### ISSUE TITLE
improvement README.md

### ISSUE THREAD

**Issue by msalcantara** [dgraph-io/badger#697]
In example `Managing transactions manually` in README.md has a line example that use Commit() func of Txn struct, and in example is passed a parameter nil, but func commit not receive any parameters.

README.md
```
// Start a writable transaction.
txn := db.NewTransaction(true)
defer txn.Discard()

// Use the transaction...
err := txn.Set([]byte("answer"), []byte("42"))
if err != nil {
    return err
}

// Commit the transaction and check for error.
if err := txn.Commit(nil); err != nil {     //should  be txt.Commit()
    return err
}
```

**Comment by manishrjain** [comment]
That API was changed recently. The func is now part of `CommitWith`.

Do you want to send a PR for txn.Commit?

**Comment by msalcantara** [comment]
I would like. Can leave it with me.

**Comment by martinmr** [comment]
Fixed by #702
